README.md

```diff
@@ -38,48 +38,74 @@ You can run the rest of the roslaunch files in the same terminal
  roslaunch px4 posix_sitl.launch
 ```
 
-### Building mavros_controllers
+You will need source the PX4 environment in every new terminal you open to launch mavros_controllers. 
+
+### Building mavros from source 
+
 
 - Create a catkin workspace
 
+a) This folder will probably be already created since the previous process would have created. If it is not present, do the following, if it is present, skip.
+
 ```bash
 mkdir -p ~/catkin_ws/src
 ```
 
-###### Download dependencies, this repository and build
+b) Now you need to remove the mavros package already present in the catkin_ws/src folder, and replace it with the source files from [mavros](https://github.com/mavlink/mavros):
+
+```bash 
+cd ~/catkin_ws/src
+rm -rf mavros
```

Comment:
So, I guess this is here because the installation scrpts also installs mavros in the workspace? I am not sure if this is generic enough. Not all people are going to use the installation script. What do you think?

Reply:
Yes, the installation script for PX4 also install mavros, so that part of the instructions is for removing that folder and allowing and ensuring the new installation from source that follows. 

However, now that I double [check it](https://raw.githubusercontent.com/PX4/Devguide/master/build_scripts/ubuntu_sim_ros_melodic.sh), the PX4 install script also install mavros from source as well, so that part seems unnecessary. Under the assumption that all users will follow the instructions from the beginning, I think this part can be removed.

Reply:
Yes, please remove this part. In my opinion, it will be a little confusing

Reply:
Just pushed the changes required and added a small fix.